common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -801,21 +800,21 @@ def test_search(self):
         self.create_teams(self.topic, 5)
         self.browse_teams_page.visit()
         events = [{
-            'event_type': 'edx.team.searched',
+            'event_type': 'edx.team.page_viewed',
             'event': {
-                'search_text': search_text,
+                'page_name': 'search-teams',
                 'topic_id': self.topic['id'],
-                'number_of_results': 0
+                'team_id': None
             }
         }, {
-            'event_type': 'edx.team.page_viewed',
+            'event_type': 'edx.team.searched',
             'event': {
-                'page_name': 'search-teams',
+                'search_text': search_text,
                 'topic_id': self.topic['id'],
-                'team_id': None
+                'number_of_results': 0
             }
         }]
-        with self.assert_events_match_during(self.only_team_events, expected_events=events):
+        with self.assert_events_match_during(self.only_team_events, expected_events=events, in_order=False):
```

Comment:
I'm not 100% sure, but my thinking is that they are both asynchronous requests and so there is no guarantee that they will actually fire the event in the same order. The test was flaky until I introduced this parameter. Note that there is no need for them to be recorded in order (as far as I know) as they will be tracked independently.